locale/en/site.json

```diff
@@ -159,10 +159,11 @@
   },
   "banners": {
     "index": {
-      "startDate": "2022-06-27T16:00:00.000Z",
-      "endDate": "2022-07-13T16:00:00.000Z",
-      "text": "New security releases now available for Node.js 14, 16, and 18 release lines",
-      "link": "https://nodejs.org/en/blog/vulnerability/july-2022-security-releases/"
+      "startDate": "2022-07-29T16:00:00.000Z",
+      "endDate": "2022-08-11T16:00:00.000Z",
+      "text": null,
+      "html": "<img alt='Grace Hopper Celebration' src='/static/images/ghc-banner.png' />",
```

Comment:
TL;DR:

```js
 "html": "<img alt='Grace Hopper Celebration' src='https://nodejs.org/static/images/ghc-banner.png' />",
```

Reply:
Can you open the PR for that?

Reply:
Here we go: https://github.com/nodejs/nodejs.org/pull/4762